Author: Daniel Lange <DLange@git.local>
Date:   Tue Feb 9 20:25:57 2021 +0100

    Make ZFS Meter "Unavailable" text match others -> FAILED_READ coloring

Author: Ross Williams <ross@ross-williams.net>
Date:   Tue Feb 9 16:59:03 2021 +0000

    [..] correctly color ZFS ARC ratio

--- a/zfs/ZfsArcMeter.c
+++ b/zfs/ZfsArcMeter.c
@@ -72,7 +72,7 @@
       RichString_appendAscii(out, CRT_colors[ZFS_OTHER], buffer);
    } else {
       RichString_writeAscii(out, CRT_colors[METER_TEXT], " ");
-      RichString_appendAscii(out, CRT_colors[FAILED_SEARCH], "Unavailable");
+      RichString_appendAscii(out, CRT_colors[FAILED_READ], "Unavailable");
    }
 }
 
--- a/zfs/ZfsCompressedArcMeter.c
+++ b/zfs/ZfsCompressedArcMeter.c
@@ -54,11 +54,11 @@
       RichString_appendAscii(out, CRT_colors[METER_VALUE], buffer);
       RichString_appendAscii(out, CRT_colors[METER_TEXT], " Compressed, ");
       ZfsCompressedArcMeter_printRatioString(this, buffer, sizeof(buffer));
-      RichString_appendAscii(out, CRT_colors[METER_VALUE], buffer);
+      RichString_appendAscii(out, CRT_colors[ZFS_RATIO], buffer);
       RichString_appendAscii(out, CRT_colors[METER_TEXT], " Ratio");
    } else {
       RichString_writeAscii(out, CRT_colors[METER_TEXT], " ");
-      RichString_appendAscii(out, CRT_colors[FAILED_SEARCH], "Compression Unavailable");
+      RichString_appendAscii(out, CRT_colors[FAILED_READ], "Compression Unavailable");
    }
 }
 
